test/transport_web/solutions/open_data_test.exs

```diff
@@ -0,0 +1,27 @@
+defmodule TransportWeb.OpenDataTest do
+  @moduledoc """
+  When the Transport team direct me to transport.beta.gouv.fr
+  I want to have an easy way to open my data on this site
+  """
+
+  use TransportWeb.ConnCase, async: true
+  use Hound.Helpers
+
+  hound_session()
+
+  @tag :integration
+  test "I'm redirect on login page when I'm not logged" do
```

Comment:
I'm **redirected to the** login page when I'm not logged **in**

Reply:
done